src/community/partnerships/facemap.md

```diff
@@ -81,17 +81,17 @@ and Facemap.
 
 ## Development Roadmap
 
-- Mechanism to import Facemap results - Completed 2022 - 
+- [x] Mechanism to import Facemap results - Completed 2022 - 
```

Comment:
Do we need completion dates now that we have check boxes? I think I would remove dates here. If someone cares to know when something was implemented, they can look at the commit history. If we decide to change, this would require edits on each other partnership page

Reply:
Good point.  Removed.